hmt_escrow/job.py

```diff
@@ -417,7 +421,10 @@ def setup(self, gas: int = GAS_LIMIT, sender: str = None) -> bool:
         reputation_oracle = str(self.serialized_manifest["reputation_oracle_addr"])
         recording_oracle = str(self.serialized_manifest["recording_oracle_addr"])
         hmt_amount = int(self.amount * 10 ** 18)
-        hmtoken_contract = get_hmtoken()
+        hmtoken_address = (
```

Comment:
It might be better to init self.hmtoken_addr in __init__, and just use self.hmtoken_addr in get_hmtoken in the line below.

Reply:
done